kolena/_experimental/search/embeddings.py

```diff
@@ -65,3 +77,61 @@ def upload_embeddings(key: str, embeddings: List[Tuple[str, np.ndarray]]) -> Non
     krequests.raise_for_status(res)
     data = from_dict(data_class=API.UploadEmbeddingsResponse, data=res.json())
     log.success(f"uploaded embeddings for key '{key}' on {data.n_samples} samples")
+
+
+def upload_dataset_embeddings(dataset_name: str, key: str, df_embedding: pd.DataFrame) -> None:
```

Comment:
do we support partial dataset embedding upload? 
what happen if they provide different model locators for the datapoint in the same dataset? do we handle it gracefully when searching the embeddings?